tests/integration/golang/aim/project/get_project_test.go

```diff
@@ -0,0 +1,53 @@
+//go:build integration
+
+package run
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/G-Research/fasttrackml/pkg/api/aim/response"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/fixtures"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/helpers"
+)
+
+type GetProjectTestSuite struct {
+	suite.Suite
+	client          *helpers.HttpClient
+	projectFixtures fixtures.ProjectFixtures
```

Comment:
can we just use a pointer here to be consistent with everything.

Reply:
done